jupyterlab_nvdashboard/apps/gpu.py

```diff
@@ -169,21 +172,292 @@ def cb():
             pynvml.nvmlDeviceGetPcieThroughput(
                 gpu_handles[i], pynvml.NVML_PCIE_UTIL_TX_BYTES
             )
-            / (1024.0 * 1024.0)  # Convert KB/s -> GB/s
+            * 1024.0
             for i in range(ngpus)
         ]
         src_dict["pci-rx"] = [
             pynvml.nvmlDeviceGetPcieThroughput(
                 gpu_handles[i], pynvml.NVML_PCIE_UTIL_RX_BYTES
             )
-            / (1024.0 * 1024.0)  # Convert KB/s -> GB/s
+            * 1024.0
             for i in range(ngpus)
         ]
         source.data.update(src_dict)
 
     doc.add_periodic_callback(cb, 200)
 
 
+def nvlink(doc):
+
+    import subprocess as sp
+
+    # Use device-0/link-0 to get "upper bound"
+    counter = 1
+    nlinks = pynvml.NVML_NVLINK_MAX_LINKS
+    nvlink_ver = pynvml.nvmlDeviceGetNvLinkVersion(gpu_handles[0], 0)
+    GB = 1024.0 * 1024.0 * 1024.0
```

Comment:
This is also a `GiB` (I know I'm being pedantic though 😄).

Reply:
You're not being pedantic :) - This was a mistake on my end